packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -955,11 +955,12 @@ class Viewport implements IViewport {
 
     // update clippingPlanes if volume viewports
     const actorEntry = this.getDefaultActor();
-    if (actorEntry?.actor?.isA('vtkVolume')) {
-      this.updateClippingPlanesForActors(updatedCamera);
-    }
 
-    if (actorEntry?.actor?.isA('vtkImageSlice')) {
+    const isImageSlice = actorEntry?.actor?.isA('vtkImageSlice');
```

Comment:
Consider defining a function that provides this check if it is a common on, eg
const actorIsA(actorEntry,'tvkImageSlice')